app/lib/liquid/lua_utility.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Liquid
```

Comment:
This is fine but it has nothing to do with the Liquid used in CMS ;)

Reply:
Moved away from the `Liquid`.